tests/dummy/app/controllers/application.js

```diff
@@ -6,6 +6,16 @@ export default Ember.Controller.extend({
   buttonStyles: "expand-left expand-right expand-up expand-down contract contract-overlay zoom-in zoom-out slide-left slide-right slide-up slide-down".w(),
   buttonColors: "green red blue mint purple".w(),
   timeout: 2E3,
+  delay: 1000,
+
+
+
+  init(){
+    this._super.apply(this, arguments);
+
+    this.set('selectedAnimation', this.get('buttonStyles').objectAt(0));
+    this.set('selectedColor', this.get('buttonColors').objectAt(0));
```

Comment:
Thanks, I didn't know that. 
btw. do you know the reason why it is inefficient? My guess is that there is no reason to notify the observers, when setting values in init() method, since the controller itself is not yet initialized - am I right?